gallery/src/styles/NxGrid/NxGridExamples.tsx

```diff
@@ -30,6 +30,12 @@ const NxGridExamples = () =>
                         codeExamples={NxGridScrollingCode}>
       An example of an <NxCode>nx-grid</NxCode> containing two cells each of which scroll. As is typical in this
       case, a horizontal keyline is added above the row of cells.
+
+      <p>
```

Comment:
The entire `GalleryExampleTile` children actually ends up in a `<p>` so this creates a nested `<p>` which is invalid.  Unfortunately `GalleryExampleTile` doesn't have a way to have multiple paragraphs of children, so generally all the text just gets stuffed in the one paragraph.

Reply:
> The entire `GalleryExampleTile` children actually ends up in a `<p>` so this creates a nested `<p>` which is invalid. Unfortunately `GalleryExampleTile` doesn't have a way to have multiple paragraphs of children, so generally all the text just gets stuffed in the one paragraph.

Ah ok. Adjusted to remove the `<p>` tag: 07622c38f0a3402540c88a01685ce52de9f49228